app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,199 @@
+$track-width: 16em;
+$track-height: .625em;
+$track-background: $white;
+$track-fill: linear-gradient(90deg, $grey-3 25%, $grey-3 100%);
+
+$thumb-diameter: 1.5em;
+
+$option-colours: (
+  "leave-pot-untouched": $color-option-leave-pot-untouched,
+  "guaranteed-income": $color-option-guaranteed-income,
+  "adjustable-income": $color-option-adjustable-income,
+  "take-cash": $color-option-take-cash,
+  "whole-pot": $color-option-take-whole-pot,
+  "mix-options": $color-option-mix-options
+);
+
+
+// MIXINS
+@mixin track($flag: true) {
+  width: $track-width;
+  height: $track-height;
+  border-radius: $track-height / 2;
+  box-shadow: inset 2px 2px 0 darken($grey-2, 1.7);
+  border-right: 2px solid $grey-3;
+  border-bottom: 2px solid $grey-3;
+  background: $track-background;
+
+  @if $flag {
+    background: $track-fill no-repeat $track-background;
+    background-size: 0 100%;
+  }
+}
+
+@mixin thumb($option: "leave-pot-untouched") {
+  width: $thumb-diameter;
+  height: $thumb-diameter;
+  border-radius: $thumb-diameter;
+  border: 3px solid map-get($option-colours, $option);
+  background: map-get($option-colours, $option);
+  cursor: ew-resize;
+}
+
+.slider {
+  width: $track-width;
+  position: relative;
+}
+
+.slider__range-input {
+  position: relative;
+  padding: 0;
+  margin: 0;
+  width: $track-width;
+  height: $thumb-diameter * 2;
+  font-size: 1em;
+  cursor: pointer;
+
+  &,
+  &::-webkit-slider-runnable-track,
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+  }
+
+  &::-moz-range-track {
+    @include track();
+  }
+
+  &::-moz-focus-outer {
+    outline: none;
+  }
+
+  &::-webkit-slider-runnable-track {
+    @include track();
+  }
+
+  &::-ms-track {
+    @include track(false);
+    border: 0;
+    color: transparent;
+  }
+
+  &::-ms-fill-lower {
+    border-radius: $track-height/2;
+    background: $track-fill;
+  }
+
+  &::-webkit-slider-thumb {
+    @include thumb();
+    position: relative;
+    margin-top: ($track-height - $thumb-diameter)/2;
+  }
+
+  &::-moz-range-thumb {
+    @include thumb();
+  }
+
+  &::-ms-thumb {
+    @include thumb();
+  }
+
+  &:focus {
+    outline: 2px solid $focus-colour;
+  }
+}
+
+.slider--leave-pot-untouched {
+  .slider__range-input {
```

Comment:
Is it possible to extract `.slider__range-input` into something that can be reused so that we don't have to duplicate it for each option variation?